.github/workflows/simple-lint.yml

```
@@ -0,0 +1,18 @@
+name: Simple Lint
+on: pull_request
+
+jobs:
+  run-linters:
+    name: simple run for linters
```

nit: Capitalize first letter
```suggestion
    name: Simple run for linters
```